foreman-bootloaders-redhat/foreman-generate-bootloaders

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+MODULES="all_video boot btrfs cat configfile echo ext2 fat font gfxmenu gfxterm gzio halt hfsplus iso9660 jpeg loadenv loopback lvm mdraid09 mdraid1x minicmd normal part_apple part_msdos part_gpt password_pbkdf2 png reboot search search_fs_uuid search_fs_file search_label serial sleep syslinuxcfg test tftp video xfs linux"
+generate() {
+        grub2-mknetdir --net-directory=/var/lib/tftpboot/ --locales="" --fonts="" -d /usr/lib/grub/$1/ --subdir=grub2 >/dev/null
+        grub2-mkimage -d /usr/lib/grub/$1/ -O $1 -o /var/lib/tftpboot/$2 -p "" $MODULES
+}
+if [[ "$1" == "aa64" ]]; then
+        [[ "$2" != "noinst" ]] && yum -y install grub2-efi-aa64-modules >/dev/null
```

Comment:
Shouldn't this be a dependency?

Reply:
Unfortunately, aa64 and ppc* are not in CentOS, hard dependency would break it. Therefore all non-intel archs are explicit yum install. The option to skip installation is there because I call the script from yum transaction, unsure if yum in yum is allowed. I will redesign the options to be more explicit.